src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,68 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline, with a given date/time/both 
+ * which indicates when the user needs to complete it by.
+ */
+class Deadline extends Task {
+    private LocalDate date;
+    private LocalTime time;
```

Comment:
Is it better to use `java.util.Date` to storage the date/time together in one field?

Reply:
Good idea, but I use them separately as I wanted more leeway for the user. They can type both date, time or both instead of just date time, unless java.util.Date is able to do that.